### ISSUE TITLE
AlertListener Error: [Errno -3]

### ISSUE THREAD

**Issue by Alan111S** [Taxel/PlexTraktSync#1561]
### Confirmation

- [X] I have read the [README.md](https://github.com/Taxel/PlexTraktSync#readme) on the project homepage
- [X] I have checked if identical issue already exists
- [ ] I have tried downgrading to find version that can be used as a workaround

### The problem

SInce a couple of days ago, I've been getting this issue in my logs. I've tried logging in to Plex and Trakt again but that hasn't resolved the issue. I run the app in Docker with command 'watch'.

I also run a cronjob to execute sync as a separate command every 15 minutes and that is working fine.

### Error trace / logs

```text
INFO     Starting AlertListener:                                                
         wss://192-168-50-200.5f356be8e49849d8b5d78804596182d2.plex.direct:32400
         /:/websockets/notifications?X-Plex-Token=<hidden>                      
ERROR    AlertListener Error: [Errno -3] Try again                              
ERROR    [Errno -3] Try again - goodbye                                         
ERROR    Server closed connection                                               
ERROR    Listener finished. Restarting in 15 seconds
```

### Expected behavior

No errors should be shown

### Steps to reproduce the behavior

docker exec -it ptsync /bin/sh

Then

plextraktsync watch

### Inspect of problematic items

```text
None
```

### Workarounds

None

### Install method

docker-compose

### Config file contents

```yaml
# Config File: /app/config/config.yml
cache:
  path: /app/config/trakt_cache
excluded-libraries:
- Demos
- MKV Kids
- MKV Yasmin
- Music
- Photo
- Video
config:
  dotenv_override: true
plex:
  timeout: 30
logging:
  append: true
  console_time: false
  debug: true
  filename: plextraktsync.log
  filter_loggers: null
  filter: null
sync:
  plex_to_trakt:
    collection: true
    clear_collected: false
    ratings: false
    watched_status: false
    watchlist: true
  trakt_to_plex:
    liked_lists: true
    ratings: false
    watched_status: true
    watchlist: false
    watchlist_as_playlist: false
  rating_priority: plex
watch:
  add_collection: false
  remove_collection: false
  scrobble_threshold: 90
  username_filter: true
  media_progressbar: true
xbmc-providers:
  movies: imdb
  shows: tvdb
```

### Version

0.26.11

### Python Version

3.11.4

### Operating System and Version

Linux

**Comment by glensc** [comment]
This is some kind of networking error in your network or docker. Nothing we can do. Especially as no software was updated in the process. You might try restarting docker daemon or rebooting the host.

**Comment by Alan111S** [comment]
Thanks for your answer. I updated my router software recently and it broke local DNS - I had to add some settings in the router to reinstate that. Also, my Docker Subnet changed from 172.16.0.0 to 172.17.0.0, so I had to create a new Route to my LAN.

Anyway it's working fine again now. Sorry to disturb !

**Comment by Alan111S** [comment]
Closed

**Comment by Alan111S** [comment]
In case anybody comes across this issue, I figured it out eventually. I edited servers.yml and made sure the urls entry listed there match my docker environment variable PLEX_ADVERTISE_URL and also the 'Custom server access URL' in Plex Network Settings.

Also, test that URL so if you use https it doesn't give certificate errors. If AlertListener is using wss, it means you specified https. If using ws, it means you specified http.

In Plex Network Settings set Secure Connections to 'Preferred' not 'Required'
